### PR TITLE
Adapter Agentos:Add chunk_hash return and expose evict/prefetch APIs

### PR DESCRIPTION
**Pull request by jianghv**
To interface with the upper-layer AgentOS, each KV Cache needs a unique identifier. Therefore, a chunk_hash field is added to the response returned by the request.

### CODE REVIEW

**Comment by jianghv** [comment]
9b440d77808532cb6f7ae0ad7c366764
include chunk_hashes field in curl request response

**Comment by matthewygf** [comment]
Hmm, I think there might be a better way to re-calculate the hash everytime when `request_finished`.

In theory, 
1. every lookup request already calculate the token hashes per chunk, and this is done at the scheduler side.
2. it is possible to cache the token hashes at the scheduler side all the way till `bind_connector_meta` and attached the chunk_hash then passes to the worker side.
3. At the worker side you can then just get from the request_tracker.

Wdyt ? @jianghv

**Review by matthewygf — CHANGES_REQUESTED** [review]
Left a cmt, on whether we should attached the hash from lookup.

**Comment by jianghv** [comment]
Previous calculations were all done in the worker process, while request_finished is called by the scheduler process, so it cannot be reused.

**Review by chloroethylene — CHANGES_REQUESTED** [review]
Add UT for new feature

**Comment by jianghv** [comment]
402b0a25e038ba6113d78db977fb639b

All the local CI gate checks pass

File: tests/v1/lookup_client/test_lmcache_ascend_async_lookup_client.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+class TestLMCacheAscendAsyncLookupClient:
+    @staticmethod
+    def _make_client(**attrs):
+        # First Party
+        from lmcache_ascend.v1.lookup_client.lmcache_ascend_async_lookup_client import (
+            LMCacheAscendAsyncLookupClient,
+        )
+
+        client = LMCacheAscendAsyncLookupClient.__new__(LMCacheAscendAsyncLookupClient)
```
Comment (chloroethylene) [inline tests/v1/lookup_client/test_lmcache_ascend_async_lookup_client.py]: why not LMCacheAscendAsyncLookupClient() directly construct a object

File: lmcache_ascend/__init__.py [hunk 2]
```diff
@@ -540,6 +530,44 @@ def _patch_rpc_utils():
...
+def _patch_storage_manager():
+    # Third Party
+    from lmcache.v1.storage_backend.storage_manager import StorageManager
+
+    # First Party
+    from lmcache_ascend.v1.storage_backend.storage_manager import (
+        patched_prefetch_all_done_callback,
+    )
+
+    StorageManager.prefetch_all_done_callback = patched_prefetch_all_done_callback
```
Comment (chloroethylene) [inline lmcache_ascend/__init__.py:565]: recommand to change `patched_prefetch_all_done_callback` to `StorageManager_prefetch_all_done_callback`

File: lmcache_ascend/v1/storage_backend/storage_manager.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: Apache-2.0
+# Third Party
+from lmcache.logging import init_logger
+from lmcache.v1.event_manager import EventStatus, EventType
+
+logger = init_logger(__name__)
+
+
+def patched_prefetch_all_done_callback(
```
Comment (chloroethylene) [inline lmcache_ascend/v1/storage_backend/storage_manager.py:9]: add a comment to describe the changed code .

File: lmcache_ascend/v1/lookup_client/lmcache_ascend_async_lookup_client.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import torch
+
+
+class LMCacheAscendAsyncLookupClient(LMCacheAsyncLookupClient):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # lookup_hashes_cache init start---------------------
+        self._lookup_hashes_cache = {}
+        self._max_cache_size = int(
+            kwargs.pop(
```
Comment (chloroethylene) [inline lmcache_ascend/v1/lookup_client/lmcache_ascend_async_lookup_client.py]: __init__ use the same interface as LMCacheAsyncLookupClient.__init__ . And use `config.xxx` to get config

**Review by matthewygf — APPROVED** [review]
LGTM

**Comment by jianghv** [comment]
1.When enable_chunk_hashes_return is True, the prefetch/evict function works correctly.
image

2.When enable_chunk_hashes_return is False, the prefetch/evict interfaces are disabled, which does not affect the original service.
image
3.CI access control verification failed only once, and this failure can pass when run separately
image
image

**Review by chloroethylene — APPROVED** [review]
